src/clang-c-frontend/clang_c_convert.h

```diff
@@ -64,6 +64,8 @@ class clang_c_convertert
   static void
   gen_typecast_to_union(exprt &dest, const typet &type, const messaget &msg);
 
+  std::string get_decl_name(const clang::NamedDecl &nd);
```

Comment:
make it `static` as before?

Reply:
I didn't manage to to use the `static std::String get_decl_name` function  in `clang_cpp_convertert` class. 

I tried `clang_c_convertert::get_decl_name(fd)`, but got linker error: 
```
/usr/bin/ld: src/clang-cpp-frontend/libclangcppfrontend.a(clang_cpp_convert.cpp.o): in function `clang_cpp_convertert::get_expr(clang::Stmt const&, exprt&)':
../src/clang-cpp-frontend/clang_cpp_convert.cpp:617: undefined reference to `clang_c_convertert::get_decl_name[abi:cxx11](clang::NamedDecl const&)'
/usr/bin/ld: src/clang-cpp-frontend/libclangcppfrontend.a(clang_cpp_convert.cpp.o): in function `clang_cpp_convertert::build_member_from_component(clang::FunctionDecl const&, exprt&)':
../src/clang-cpp-frontend/clang_cpp_convert.cpp:704: undefined reference to `clang_c_convertert::get_decl_name[abi:cxx11](clang::NamedDecl const&)'
```

Hence, changed it to a public function.

Reply:
Public is fine; I meant
```diff
diff --git a/src/clang-c-frontend/clang_c_convert.h b/src/clang-c-frontend/clang_c_convert.h
index b8f9560e1e..8c1c43ae51 100644
--- a/src/clang-c-frontend/clang_c_convert.h
+++ b/src/clang-c-frontend/clang_c_convert.h
@@ -64,7 +64,7 @@ public:
   static void
   gen_typecast_to_union(exprt &dest, const typet &type, const messaget &msg);
 
-  std::string get_decl_name(const clang::NamedDecl &nd);
+  static std::string get_decl_name(const clang::NamedDecl &nd);
 
 protected:
   clang::ASTContext *ASTContext;
```